galaxyproject/galaxy
# [22.05] Minor updates to Collection Progress Bar

To fix #14131  and #14136 
I moved the value of the items to tooltips, and set the progress bar to only be visible when there are items that are not in a terminal state (ok/error)

https://user-images.githubusercontent.com/26912553/174950469-55489fe6-3169-4ebd-a23b-6d35ad1db01c.mp4



## How to test the changes?
(Select all options that apply)
  1. Run a slower tool over a collection, such as "Concatenate datasets (with sleep)" and/or choose a large collection
  2. View the collection progress bar
  3. Note that the numbers no longer appear over the states, and instead are visible in a tooltip

## License
